aivpn.py

```diff
@@ -5,48 +5,36 @@
 
 import re
 import sys
-import json
 import datetime
 import argparse
 import logging
-import configparser
 from common.database import *
 
-def manage_info(REDIS_CLIENT,profile_name):
+
+def manage_info(REDIS_CLIENT, profile_name):
     """
     Retrieve information about an AI VPN profile_name
     """
-    profile_information = {}
+
     try:
         logging.debug('Manage info: {profile_name}')
-        vpn_type = get_profile_vpn_type(profile_name,REDIS_CLIENT)
+        vpn_type = get_profile_vpn_type(profile_name, REDIS_CLIENT)
         profile_creation_time = 'n/a'
         profile_expiration_time = 'n/a'
         profile_reported_time = 'n/a'
         profile_deletion_time = 'n/a'
 
-        if exists_active_profile(profile_name,REDIS_CLIENT):
-            profile_creation_time = datetime.datetime.fromtimestamp(float(get_active_profile_creation_time(profile_name,REDIS_CLIENT)))
+        if exists_active_profile(profile_name, REDIS_CLIENT):
+            profile_creation_time = datetime.datetime.fromtimestamp(
+                float(get_active_profile_creation_time(profile_name, REDIS_CLIENT)))
             profile_active = 'active'
         else:
             profile_active = 'expired'
-            profile_information = json.loads(get_expired_profile_information(profile_name,REDIS_CLIENT))
-            try:
```

Comment:
Here's an example of one case where it would break without the try/except:
```bash
python3 aivpn.py --redis "172.16.1.3" manage --info 20221107010044-naive_obedient
Exception in manage_info: could not convert string to float: ''
```

Happy to hear how to do it better but still catching cases like this.

Reply:
sounds good. once you back we can look into it 👍